screens/LogoutScreen.js

```diff
@@ -1,33 +1,32 @@
 import React from 'react';
-import { View, StyleSheet, Button} from 'react-native';
+import { StyleSheet, Button } from 'react-native';
+
+import Colors from '../utils/colors';
+import SafeView from '../components/SafeView';
 import useStatusBar from '../hooks/useStatusBar';
 import { logout } from '../components/Firebase/firebase';
-import {NavigationContainer} from '@react-navigation/native';
-import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
-import {Image} from 'react-native';
-import Colours from '/Users/sandra/Maiagram-expo-firebase/screens/res/Colours.js'
-
-
-
-const Tab = createBottomTabNavigator();
-
 
 export default function LogoutScreen() {
   useStatusBar('dark-content');
+
   async function handleSignOut() {
     try {
       await logout();
     } catch (error) {
       console.log(error);
     }
   }
-  return (
-
-    <Tab.Navigator>
-      <Tab.Screen name="Sign Out" component={handleSignOut} />
 
-      
-    </Tab.Navigator>
-
-);
-}
\ No newline at end of file
+  return (
+    <SafeView style={styles.container}>
```

Comment:
The tab navigation was expecting a view component instead another tab navigator and that's what was producing the error. If you wants the navigation tab acts as a button, I supose we can create a renderless component that trigger the sign out action once rendered